modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -162,17 +163,18 @@ spec:
 <5> Specify system config options for the `ptp4l` service, for example `-2`. The options should not include the network interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because the network interface name and the service config file are automatically appended.
 <6> Specify the needed configuration to start `ptp4l` as boundary clock. For example, `ens1f0` synchronizes from a grandmaster clock and `ens1f3` synchronizes connected devices.
 <7> The interface that receives the synchronization clock.
-<8> The interface that synchronizes downstream connected devices.
-<9> Specify system config options for the `phc2sys` service, for example `-a -r`. If this field is empty the PTP Operator does not start the `phc2sys` service.
-<10> Scheduling policy for ptp4l and phc2sys processes. Default value is `SCHED_OTHER`. Use `SCHED_FIFO` on systems that support FIFO scheduling.
-<11> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`.
-<12> Optional. If `ptpClockThreshold` stanza is not present, default values are used for `ptpClockThreshold` fields. Stanza shows default `ptpClockThreshold` values.
-<13> Specify an array of one or more `recommend` objects that define rules on how the `profile` should be applied to nodes.
-<14> Specify the `profile` object name defined in the `profile` section.
-<15> Specify the `priority` with an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. If a node can be matched with multiple profiles according to rules defined in the `match` field, the profile with the higher priority is applied to that node.
-<16> Specify `match` rules with `nodeLabel` or `nodeName`.
-<17> Specify `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command.
-<18> Specify `nodeName` with `node.Name` from the node object by using the `oc get nodes` command.
+<8> For Intel Columbiaville 800 Series NICs, set `boundary_clock_jbod` to `0`. For Intel Fortville NICs, set `boundary_clock_jbod` to `1`.
```

Comment:
Sorry, but I still don't see why this is true in all cases. If, for instances, you have a Fortville NIC with two ports, one acting as MASTER and the other acting as SLAVE, then this option must be set to zero. Am I right? So this makes sense whenever you have a node acting as a BC using two different NICs having their PHCs synced through the phc2sys program. Otherwise, this should be always leave as default (zero) value.

Reply:
@aneeshkp @josephdrichard thoughts?